src/userCLI/src/test/kotlin/org/imperial/mrc/hint/userCLI/AppTests.kt

```diff
@@ -0,0 +1,68 @@
+package org.imperial.mrc.hint.userCLI
+
+import org.imperial.mrc.hint.exceptions.UserException
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.ApplicationContext
+
+//The hint db container must be running to run these tests
+@ActiveProfiles(profiles=["test"])
```

Comment:
Picks up application-test.properties to augment/overwrite application.properties